modules/apps/29-fee/ibc_module_test.go

```diff
@@ -709,8 +708,7 @@ func (suite *FeeTestSuite) TestOnTimeoutPacket() {
 
 				expectedBalance = originalBalance.
 					Add(identifiedFee.Fee.RecvFee[0]).
-					Add(identifiedFee.Fee.AckFee[0]).
-					Add(ibctesting.TestCoin) // timeout refund for ics20 transfer
+					Add(identifiedFee.Fee.AckFee[0])
```

Comment:
I don't understand how this was passing before. Seems like you're actually changing the test logic rather than just a refactor?